File: Treinamento/Java/Estrutura_sequencial/Questão07.java
import java.util.Scanner;
public class Questão07 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite o lado do quadrado: ");
        int l = s.nextInt();
        
        double area = Math.pow(l, 2);

        System.out.printf("Área do quadrado: %.0f\n", area); 
        System.out.printf("Dobro da área: %.0f\n", area * 2);
   }
}

File: Treinamento/C/Estrutura_sequencial/Questão07.c
#include <stdio.h>
#include <math.h>

int main() {
    float lado;

    printf("Digite a medida do lado do quadrado: ");
    scanf("%f", &lado);

    printf("Área: %fcm * %fcm = %fcm² * 2 = %f\n", lado, lado, pow(lado, 2), pow(lado, 2) * 2);    
}

